packages/docs/page-config/ui-elements/skip-link/index.ts

```diff
@@ -0,0 +1,20 @@
+import apiOptions from "./api-options";
+import apiDescription from './api-description';
+
+export default definePageConfig({
+  blocks: [
+    block.title("Skip Link"),
+    block.paragraph("`va-skip-link` skip to the target component."),
```

Comment:
We need more text here. You can describe in technical terms what component does - then ask chat gpt to get you some text.